src/util/bv_encoding.h

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************\
+
+Module: bv_encoding configuration
+
+Author: Diffblue Ltd
+
+\*******************************************************************/
+
+#ifndef CPROVER_UTIL_BV_ENCODING_H
+#define CPROVER_UTIL_BV_ENCODING_H
+
+#include <cstddef>
+
+struct bv_encodingt
```

Comment:
Moves it into a different namespace though

Reply:
Ok. I'll leave it where it was then.